src/_playground/Routes.js

```diff
@@ -33,155 +37,309 @@ import { TimePickerComponent } from '../TimePicker/TimePicker.Component';
 import { ToggleComponent } from '../Toggle/Toggle.Component';
 import { TokenComponent } from '../Token/Token.Component';
 import { TreeComponent } from '../Tree/Tree.Component';
+import Usage from './documentation/Markdown/Usage.md';
 import { BrowserRouter, NavLink, Redirect, Route, Switch } from 'react-router-dom';
 import React, { Component } from 'react';
 
+
+
 export default class Routes extends Component {
     constructor(props) {
         super(props);
         this.state = {
             routes: [
+                {
```

Comment:
Is there a reason why these are in state?  I don't believe they ever morph so couldn't it just be a simple `const`?

Reply:
no reason, I just left them where I found them. I'll change it